README.md

```diff
@@ -22,8 +22,13 @@ This system allows users to view, create, edit, and delete a list of: users, pro
 - Run ```npm install``` from the root of the directory to install all of the dependencies
 
 ## Get Started
-- Set up the database using ```npm run db:reset``` command in terminal
+- Make a config.json file in config folder located in the root directory. Use config.json.example template.
+- Make a .env file in the root directory. Use the .env.example template also located in the root directory.
```

Comment:
change to something like "duplicate the .env.example file and change the NODE_ENV to match and environment used in the config.json"